Algorithm-Pig-Latin.md

```diff
@@ -117,40 +157,33 @@ translatePigLatin("consonant");
 
 :rocket: [Run Code](https://repl.it/CLmv/0)
 
-### Fourth Solution (recursive and declarative)
```

Comment:
True! Alright, I'll make the necessary changes. Should I put this as the Basic solution?